app/src/main/java/agency/five/codebase/android/movieapp/ui/favorites/mapper/FavoritesMapperImpl.kt

```diff
@@ -0,0 +1,34 @@
+package agency.five.codebase.android.movieapp.ui.favorites.mapper
+
+import agency.five.codebase.android.movieapp.model.Movie
+import agency.five.codebase.android.movieapp.ui.component.MovieCardViewState
+import agency.five.codebase.android.movieapp.ui.favorites.FavoritesMovieViewState
+import agency.five.codebase.android.movieapp.ui.favorites.FavoritesViewState
+
+class FavoritesMapperImpl : FavoritesMapper {
+    override fun toFavoritesViewState(favoriteMovies: List<Movie>): FavoritesViewState {
+
+        val favoriteMoviesViewStates = mutableListOf<FavoritesMovieViewState>()
+
+        for (movie in favoriteMovies) {
```

Comment:
you can use `.map` operator on the list to convert it to the list of some other items